gettext_compile.py

```diff
@@ -8,6 +8,6 @@
 
 if __name__ == "__main__":
     do_compile("spyderlib")
-    do_compile("p_pylint", "spyderplugins")
-    do_compile("p_profiler", "spyderplugins")
-    do_compile("p_breakpoints", "spyderplugins")
+    do_compile("p_pylint", "spyderplugins/p_pylint")
+    do_compile("p_profiler", "spyderplugins/p_pylint")
+    do_compile("p_breakpoints", "spyderplugins/p_pylint")

```

Comment:
Please rebase and re-run again. I also updated a couple of translations that were failing on `spyderlib` :-)